ChiModules/KEigenvalueSolver/initialize_precursors.cc

```diff
@@ -23,7 +23,8 @@ void KEigenvalue::Solver::InitializePrecursors()
   for (auto& cell : grid->local_cells)
   {
     // ----- Cell information
-    auto xs = material_xs[matid_to_xs_map[cell.material_id]];
+    const auto xs_id = matid_to_xs_map[cell.material_id];
+    auto xs = material_xs[xs_id];
```

Comment:
This is a silly one but if you "aim to please" then ... you know.

`auto xs` makes a copy of the smart-pointer. Rather use `auto& xs` to use the pointer by reference.